src/io/flutter/FlutterUtils.java

```diff
@@ -89,7 +91,18 @@ public static boolean isDartFile(@NotNull VirtualFile file) {
   }
 
   public static boolean isAndroidStudio() {
-    return IdeInfo.getInstance().isAndroidStudio();
+    try {
+      // Try to use `IdeInfo` first by reflecting on it.
```

Comment:
Can you add a comment here that this class should be available either if the user has Android plugin installed or if they are using Android studio?